### ISSUE TITLE
Custom commands can be created with the same name as core fin commands

### ISSUE THREAD

**Issue by jhedstrom** [docksal/docksal#1054]
**Description**

I was working on a project that had a `build` command located in the `.docksal/commands` directory. The custom command's help text was properly displayed when `fin` was typed.

However, when one went to run `fin build`, it executes the actual build command that fin has.

I wonder if some checking for existent fin commands (eg, non-custom ones like `start`, `build`, `project`, etc) could be done and then at least print a warning or something.

**Steps to reproduce the issue:**

1. Put a custom command script in `.docksal/commands/build` with some help text
2. Type `fin help build` and note the custom help text is used
3. Run `fin build --help` and note the core command is executed.

**Comment by achekulaev** [comment]
I will be frank here, this item is of *very* low priority. It is medium effort (to maintain that list of commands somewhere else apart from the main `case` condition) and very low gain, in 4 years you are the first one to actually report this, and if I am not mistaken you understood what is happening within minutes even without the warning.

**Comment by jhedstrom** [comment]
Yeah, it makes sense that this would be a low priority. I hadn't actually dug into what work would be necessary to provide such a warning.
